smurfs/src/actions/smurfActions.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+export const FETCH_SMURF_DATA_START = 'FETCH_SMURF_DATA_START';
+export const FETCH_SMURF_DATA_SUCCESS = 'FETCH_SMURF_DATA_SUCCESS';
+export const FETCH_SMURF_DATA_FAILURE = 'FETCH_SMURF_DATA_FAILURE';
+export const SEND_SMURF_DATA_SUCCESS = 'SEND_SMURF_DATA_SUCCESS';
+
+export const sendData = () => {
+    return dispatch => {
```

Comment:
try and refactor this to work with context api if you want to challenge yourself